ui/raidboss/emulator/EmulatorCommon.ts

```diff
@@ -0,0 +1,127 @@
+import { LocaleNetRegex } from '../../../resources/translations';
+import NetRegexes from '../../../resources/netregexes';
+
+// Disable ban-types for cloneData as it needs to work on raw objects for performance reasons.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type DataType = { [key: string]: any } | null;
+
+export default class EmulatorCommon {
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
```suggestion
```

I think encapsulating the `any` inside of `DataType` prevents errors here already without an explicit disable.

Reply:
Yeah, this and the one above were leftover from trying to get this to pass eslint, missed cleaning them up after defining the `DataType` type. Removed for next commit.